tests/core/full_node/stores/test_coin_store.py

```diff
@@ -105,7 +105,7 @@ async def test_basic_coin_store(db_version: int, softfork_height: uint32, bt: Bl
                 else:
                     tx_removals, tx_additions = [], []
 
-                assert block.get_included_reward_coins() == should_be_included_prev
+                assert set(block.get_included_reward_coins()) == should_be_included_prev
```

Comment:
Do we want this to be order agnostic?

Reply:
well, I don't think the intention of this test ever was to ensure it predicts the order of the reward coins in the list. Maybe we could make it do that to ensure it's consistently what we expect.